sdk/python/feast/repo_config.py

```diff
@@ -93,6 +93,9 @@ class BigQueryOfflineStoreConfig(FeastBaseModel):
     dataset: StrictStr = "feast"
     """ (optional) BigQuery Dataset name for temporary tables """
 
+    project: Optional[StrictStr] = None
+    """ (optional) BigQuery project name for temporary tables """
```

Comment:
But wouldn't this also be for permanent tables? `historical_` tables could be permanent right? I think we can just drop `for temporary tables` and make it `GCP project name used for the BigQuery offline store` `BigQuery dataset name used for the BigQuery offline store`

What do you think?

Reply:
Yes you're right, those are permanent at the end. The user should choose how they want to structure their projects (with the retention policy they want etc.)

Reply:
Ok, great.

Reply:
Done I rename the new field for `project_id` and updated the description